meta-lmp-base/recipes-containers/nerdctl/nerdctl/0001-extend-ps-output.patch

```diff
@@ -0,0 +1,137 @@
+/iff --git a/src/import/cmd/nerdctl/compose_ps.go b/src/import/cmd/nerdctl/compose_ps.go
```

Comment:
did you test this build? I think there is a typo in `/iff` and bitbake will fail applying the patch

Reply:
Yes, I tested it, somehow bitbake doesn't fail,
however it's not correct/typo, i'll fix it.